clang/test/CodeGen/rtsan_insert_at_entry.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -fsanitize=realtime -emit-llvm -o - %s | FileCheck %s
```

Comment:
`*-unknown-linux-gnu` is more popular.

`none` could also be used for the OS component and is slightly misleading.